Ironmon-Tracker.lua

```diff
@@ -418,6 +420,29 @@ function Main.GenerateNextRom()
 	}
 end
 
+function Main.SaveCurrentRom(filename)
+	if Main.FileExists(filename) then
+		Main.CopyFile(filename, string.format("prev-seed %s", filename))
+		Main.CopyFile(string.format("%s.log", filename), string.format("prev-seed %s.log", filename))
```

Comment:
Two small requests here:

1) Can we change format of the previous-file by replacing the "AutoRandomized" portion of the file name with a "PreviousAttempt" label? Should be doable with a simple string `gsub()`.
   - For example, my original ROM is named "FireRed AutoRandomized.gba" and the resulting previous file name is "FireRed PreviousAttempt.gba".
2) Can you add the "PreviousAttempt" text to `Constants.Files.PostFixes`, so all of these file labels can be kept in one place?

Am interested in discussing this further if you have a preference for one method of labeling versus the other.

Reply:
[3b95fac](https://github.com/besteon/Ironmon-Tracker/pull/283/commits/3b95fac823ebcce95e9f0545fec5dc55c4a35b5e)